api/operator_ent_test.go

```diff
@@ -0,0 +1,28 @@
+//go:build ent
+// +build ent
+
+package api
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestOperator_LicenseGet(t *testing.T) {
+	t.Parallel()
+	c, s, _ := makeACLClient(t, nil, nil)
+	defer s.Stop()
+
+	operator := c.Operator()
+
+	// Make atuhenticated request.
+	_, _, err := operator.LicenseGet(nil)
+	require.NoError(t, err)
+
+	// Make unatuhenticated request.
+	c.SetSecretID("")
+	_, _, err = operator.LicenseGet(nil)
+	require.Error(t, err)
+	require.Contains(t, err.Error(), "403")
```

Comment:
Is there anyway this could return something other than a 403?

Reply:
Hum...It would return a `204` if the test runs against an OSS cluster, that's why I added the `ent` tag. But then the first test case would fail first 😅 

This check is to make sure the error we get is the one we expect. If it gets _another_ error than it's probably something that we should look into, so failing the test seem like a good thing?

Reply:
Ah, ok.  Thanks for the explanation!